src/main/java/cryptator/CryptaOperator.java

```diff
@@ -35,7 +35,8 @@ public enum CryptaOperator {
     LE("<=", (a, b) -> toBigInt(a.compareTo(b) <= 0), (a, b) -> a.le(b)),
     GE(">=", (a, b) -> toBigInt(a.compareTo(b) >= 0), (a, b) -> a.ge(b)),
 
-    AND(";", (a, b) -> toBigInt(!a.equals(BigInteger.ZERO) && !b.equals(BigInteger.ZERO)), (a, b) -> ((ReExpression) a).and((ReExpression) b));
+    AND(";", (a, b) -> toBigInt(!a.equals(BigInteger.ZERO) && !b.equals(BigInteger.ZERO)), (a, b) -> ((ReExpression) a).and((ReExpression) b)),
+    AND2("&&", (a, b) -> toBigInt(!a.equals(BigInteger.ZERO) && !b.equals(BigInteger.ZERO)), (a, b) -> ((ReExpression) a).and((ReExpression) b));
 
```

Comment:
I understand why you do this. 
In the test you are able to do parse("EX1 ; EX2 & EX3").toInfix() = "EX1 ; EX2 & EX3"
But I'm asking if it is a bit misleading to have a second AND in the ENUM.
In my optinion, I would rather keep only one version of it. 
Therefore, in the parser I would "force" the "&" to be the ";" symbol (or vice versa). 
For example, instead of doing $AND.getText() I would maybe do ";" (or equivalently "&")

Reply:
I actually hadn't thought of that, so I changed it.